config/initializers/pusher.rb

```diff
@@ -1,4 +1,3 @@
-require "pusher"
 require "pusher/push_notifications"
```

Comment:
What does this line? should it be removed?

Reply:
no, we can't because Pusher::Notifications comes from the pusher-notifications gem which is not autoloaded. I did try, stuff exploded, put it back :D